demos/object_detection_demo/python/README.md

```diff
@@ -273,6 +278,21 @@ To avoid disk space overrun in case of continuous input stream, like camera, you
 
 >**NOTE**: Windows\* systems may not have the Motion JPEG codec installed by default. If this is the case, you can download OpenCV FFMPEG back end using the PowerShell script provided with the OpenVINO &trade; install package and located at `<INSTALL_DIR>/opencv/ffmpeg-download.ps1`. The script should be run with administrative privileges if OpenVINO &trade; is installed in a system protected folder (this is a typical case). Alternatively, you can save results as images.
 
+## Running with OpenVINO Model Server
+
+You can also run this demo with model served in [OpenVINO Model Server](https://github.com/openvinotoolkit/model_server). Refer to [`OVMSAdapter`](../../common/python/openvino/model_zoo/model_api/adapters/ovms_adapter.md) to learn about running demos with OVMS.
+
+Exemplary command:
+
+```sh
+python3 object_detection_demo.py \
+  -i <path_to_video>/inputVideo.mp4 \
+  -m localhost:9000/models/object_detection \
+  -at ssd \
+  --labels <omz_dir>/data/dataset_classes/voc_20cl_bkgr.txt \
+  --adapter ovms
+```
+
```

Comment:
Shouldn't such section be in all readmes for supported demos?

Reply:
So far we were thinking about getting the change ready and approved for one demo and then propagate it to all supported demos. Once we agree on the format of ovms adapter section in one demo readme, we can copy & paste it to others. Are you okay with such approach?

Reply:
@mzegla yeah, it's ok.